Makefile

```diff
@@ -10,6 +10,7 @@ CODENAME:="$(shell . /etc/os-release; echo "$$VERSION_CODENAME")"
 -include .fips-env
 ifdef SNAP_FIPS_BUILD
     export SNAP_FIPS_BUILD
+    export SNAP_BUILD_NAME
```

Comment:
why not always do the export? and if not, the comment right before `-include` needs to be updated